src/ui-kit/experimental/hierarchical/autocomplete/autocomplete.component.ts

```diff
@@ -252,6 +251,10 @@ export class SamHierarchicalAutocompleteComponent {
     }
   }
 
+  /**
+   * Checks if the new search string matches the old search string
+   * @param searchString 
```

Comment:
If you don't have a description for the param, you can leave it out. Typescript should give us type information for the params.